setup.py

```diff
@@ -51,7 +50,7 @@
     extras_require = extras_require,
     setup_requires = ['setuptools_scm'],
```

Comment:
Running `make test` failed for me:
```
$ make test
python setup.py test
usage: setup.py [global_opts] cmd1 [cmd1_opts] [cmd2 [cmd2_opts] ...]
   or: setup.py --help [cmd1 cmd2 ...]
   or: setup.py --help-commands
   or: setup.py cmd --help
error: invalid command 'pytest'
make: *** [Makefile:21: test] Error 1
```
It looks like `setup.py` is not set up to handle the `test` (equivalently `pytest`) argument. To do that, I believe you have to make these changes (refer to [here](https://doc.pytest.org/en/3.0.4/goodpractices.html#integrating-with-setuptools-python-setup-py-test-pytest-runner)):
```suggestion
    setup_requires = ['setuptools_scm', 'pytest-runner'],
    tests_require = ['pytest'],
```
After this change, `make test` worked for me.

Reply:
Nice catch, that's what I get for reusing the same environment (which had `pytest-runner` installed) for development.